### PR TITLE
feat(replay): Let the a11y analysis run at any point inside the replay

### PR DESCRIPTION
**Pull request by ryan953**
This deserves a video, but for now here's some screen shots

Related to: https://github.com/getsentry/sentry/issues/55501

https://github.com/getsentry/sentry/assets/187460/a80c6e93-6904-427c-8949-93a41abf6fa6

### CODE REVIEW

File: static/app/views/replays/detail/accessibility/accessibilityRefetchBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          })}
+        </Flex>
+        <Button
+          size="xs"
+          priority="primary"
+          onClick={handleClickRefetch}
+          disabled={currentTime === lastOffsetMs}
+        >
+          {isPlaying
+            ? tct('Pause and Run validation for [now]', {now})
```
Comment (michellewzhang) [inline static/app/views/replays/detail/accessibility/accessibilityRefetchBanner.tsx:65]:
```suggestion
            ? tct('Pause and run validation for [now]', {now})
```

super nit

**Review by michellewzhang — APPROVED** [review]

File: static/app/views/replays/detail/accessibility/accessibilityRefetchBanner.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/views/replays/detail/accessibility/accessibilityRefetchBanner.tsx:65]: well damnit. the auto-merge got it

Comment (michellewzhang) [reply static/app/views/replays/detail/accessibility/accessibilityRefetchBanner.tsx:65]: 😭

Comment (ryan953) [reply static/app/views/replays/detail/accessibility/accessibilityRefetchBanner.tsx:65]: https://github.com/getsentry/sentry/pull/61090

**Pull request merged**